### DISCUSSION TITLE
Can you use @stdlib/plot in node.js?

### DISCUSSION THREAD

**Discussion by mhkeller** [stdlib-js/stdlib#1951] [Q&A] [answered]
Reading the source, it seems that the output of plot is an HTML tree and the user is left to render it on their own. Is that correct? The docs show a screenshot of a nice modal window appearing next to a script file. Is this meant to be loaded in the browser and the user is responsible for creating a child window and injecting the generated HTML?

**Comment by kgryte** [comment, marked as answer]
@mhkeller Yes, you can generate a view in Electron or the browser using the `view` API (see https://github.com/stdlib-js/stdlib/tree/c39cc7222bf86a5217a4fc5333b007f90f1709cd/lib/node_modules/%40stdlib/plot/ctor#plotprototypeview-viewer-).

When invoking that method, you can provide a "viewer" (as documented in https://github.com/stdlib-js/stdlib/tree/c39cc7222bf86a5217a4fc5333b007f90f1709cd/lib/node_modules/%40stdlib/plot/ctor#plotprototypeviewer).

You can also check out the examples in the `examples` folder which, IIRC, generate such views (e.g., https://github.com/stdlib-js/stdlib/blob/c39cc7222bf86a5217a4fc5333b007f90f1709cd/lib/node_modules/%40stdlib/plot/ctor/examples/scatterrug).
